src/Traits/Filter.php

```diff
@@ -19,17 +19,25 @@ trait Filter
 
     public function clearFilter(string $field = ''): void
     {
-        $this->search = '';
-
         unset($this->enabledFilters[$field]);
         unset($this->filters['number'][$field]);
+        unset($this->filters['number_start'][$field]);
+        unset($this->filters['number_end'][$field]);
         unset($this->filters['input_text'][$field]);
         unset($this->filters['boolean'][$field]);
         unset($this->filters['input_text_options'][$field]);
-
-        $this->filters = [];
+        unset($this->filters['input_date_picker'][$field]);
+        unset($this->filters['date_picker'][$field]);
+        unset($this->filters['select'][$field]);
+        unset($this->filters['multi_select'][$field]);
```

Comment:
I tested your branch, only the filter variable is blocking to follow. It should be like before, do you agree?

```
unset($this->enabledFilters[$field]);
$this->filters = [];
```

Reply:
No, it main bug. It remove all filters instead of remove one selected

Reply:
For me it is not working, if you do the same test in the browser that is in the pest test, it will give the same error

Reply:
The problem happens with tables that use joins with fields of this type: `dishes.name`. The proposed unset does not clean. Could you clean the first part of the array? 

unset($this->filters['input_text'][$field])